tempodb/encoding/vparquet2/test-data/single-tenant/b27b0e53-66a0-4505-afd6-434ae3cd4a10/meta.json

```diff
@@ -1 +1 @@
-{"format":"vParquet2","blockID":"b27b0e53-66a0-4505-afd6-434ae3cd4a10","minID":"AAAAAAAAAAAAR0votDRJ+w==","maxID":"AAAAAAAAAAD/+S7r9o+CMA==","tenantID":"single-tenant","startTime":"2022-07-04T11:11:09Z","endTime":"2022-07-04T11:11:35Z","totalObjects":134,"size":63667,"compactionLevel":0,"encoding":"none","indexPageSize":0,"totalRecords":1,"dataEncoding":"","bloomShards":1,"footerSize":8265}
\ No newline at end of file
+{"format":"vParquet3","blockID":"b27b0e53-66a0-4505-afd6-434ae3cd4a10","minID":"AAAAAAAAAAAAR0votDRJ+w==","maxID":"AAAAAAAAAAD/+S7r9o+CMA==","tenantID":"single-tenant","startTime":"2022-07-04T11:11:09Z","endTime":"2022-07-04T11:11:35Z","totalObjects":134,"size":63667,"compactionLevel":0,"encoding":"none","indexPageSize":0,"totalRecords":1,"dataEncoding":"","bloomShards":1,"footerSize":8265}
```

Comment:
I wouldn't expect this to be required?  I think we want to keep support for reading vp2, but just disallow creating new ones with it.